src/tools/annotation/LengthTool.js

```diff
@@ -228,11 +228,12 @@ export default class LengthTool extends BaseAnnotationTool {
     function textBoxText(data, rowPixelSpacing, colPixelSpacing) {
       // Set the length text suffix depending on whether or not pixelSpacing is available
       let suffix = ' mm';
-
       if (!rowPixelSpacing || !colPixelSpacing) {
         suffix = ' pixels';
       }
 
+      data.suffix = suffix;
```

Comment:
I think it would be better to keep measurement units in `data.unit` instead of `data.suffix` without the leading white space, so it will be more flexible to use in the application level.

Reply:
Done